common.mk

```diff
@@ -18,15 +18,18 @@ ifeq ($(TARGET),rhel7)
 	SKIP_SQUASH ?= 0
 	OS := rhel7
 	DOCKERFILE ?= Dockerfile.rhel7
+	DG_CONF := rhel-7-x86_64.yaml
 else ifeq ($(TARGET),fedora)
 	OS := fedora
 	DOCKERFILE ?= Dockerfile.fedora
+	DG_CONF := fedora-27-x86_64.yaml
```

Comment:
Whe is there `:=` when in other cases we use `?=` ?
Same for rhel7.

Reply:
You are right, this should be consistent.